### PR TITLE
Updated adx key name:

### PR DESCRIPTION
**Pull request by johnfolly**
- added `adx_` prefix
- updated all related commands

### CODE REVIEW

File: Babylon/groups/azure/groups/adx/commands/list_scripts.py [hunk 1]
```diff
@@ -15,14 +15,14 @@
...
+@require_deployment_key("adx_database_name", "adx_database_name")
 @timing_decorator
-def list_scripts(kmc: KustoManagementClient, cluster_name: str, resource_group_name: str, database_name: str):
+def list_scripts(kmc: KustoManagementClient, adx_cluster_name: str, resource_group_name: str, adx_database_name: str):
     """List scripts on the database"""
     r = kmc.scripts.list_by_database(resource_group_name=resource_group_name,
-                                     cluster_name=cluster_name,
-                                     database_name=database_name)
+                                     adx_cluster_name=adx_cluster_name,
+                                     adx_database_name=adx_database_name)
```
Comment (o-reo) [inline Babylon/groups/azure/groups/adx/commands/list_scripts.py:26]: the `list_by_database` function changed or you did you make a mistake ?

File: Babylon/groups/azure/groups/adx/commands/run_script.py [hunk 2]
```diff
@@ -36,11 +36,11 @@ def run_script(kmc: KustoManagementClient, cluster_name: str, resource_group_nam
...
     with open(script_file) as _script_file:
         logger.info(f"Reading {script_file}")
-        script_content = _script_file.read().replace("<database name>", database_name)
+        script_content = _script_file.read().replace("<database name>", adx_database_name)
         logger.info("Sending script to database.")
         s = kmc.scripts.begin_create_or_update(resource_group_name=resource_group_name,
-                                               cluster_name=cluster_name,
-                                               database_name=database_name,
+                                               adx_cluster_name=adx_cluster_name,
+                                               adx_database_name=adx_database_name,
```
Comment (o-reo) [inline Babylon/groups/azure/groups/adx/commands/run_script.py:43]:
```suggestion
                                               cluster_name=adx_cluster_name,
                                               database_name=adx_database_name,
```
It's probably a mistake ?

File: Babylon/groups/azure/groups/adx/commands/list_scripts.py (hunk 1 above)
Comment (o-reo) [inline Babylon/groups/azure/groups/adx/commands/list_scripts.py:26]:
```suggestion
                                     cluster_name=adx_cluster_name,
                                     database_name=adx_database_name)
```

**Review by o-reo — COMMENTED** [review]
Prbably a couple of mistakes, maybe not, tell me

**Comment by johnfolly** [comment]
> Prbably a couple of mistakes, maybe not, tell me

I did a mistake, i just replace all occurences 😅 Thank you @o-reo

**Review by o-reo — APPROVED** [review]
👍

**Pull request merged**